src/stats/stats.jl

```diff
@@ -49,25 +49,51 @@ function ExecutionStats{T}(status :: Symbol; solved :: Bool=false, tired :: Bool
   return ExecutionStats(status, solved, tired, stalled, x, f, normg, c, iter, deepcopy(eval), t, solver_specific)
 end
 
-import Base.show
+import Base.show, Base.print, Base.println
 
 function show(io :: IO, stats :: ExecutionStats)
+  show(io, "Execution stats: $(getStatus(stats))")
+end
+
+# TODO: Expose NLPModels dsp in nlp_types.jl function print
+function disp_vector(io :: IO, x :: Vector)
+  if length(x) == 0
+    @printf(io, "∅")
+  elseif length(x) <= 5
+    Base.show_delim_array(io, x, "", " ", "", false)
+  else
+    Base.show_delim_array(io, x[1:4], "", " ", "", false)
+    @printf(io, " ⋯ %s", x[end])
+  end
+end
```

Comment:
@dpo, this now prints like `NLPModels`. Inline, maximum of five elements. The tests also test the `io` now, and a different function for printing vectors.